avocado/core/runner.py

```diff
@@ -187,10 +187,9 @@ def _tick(self):
                 self.status = msg
                 self.job._result_events_dispatcher.map_method('test_progress',
                                                               False)
-                if msg['paused']:
-                    reason = msg['paused_msg']
-                    if reason:
-                        self.job.log.warning(reason)
+                paused_msg = msg['paused']
+                if paused_msg:
+                    self.job.log.warning(paused_msg)
```

Comment:
Yes, I definitely prefer a single variable to 2 variables, but would you please split the commits? It's not related to the other part where you change and limit the number of shared attributes.

Reply:
I see this change as a cohesive unit.  The goal is to reduce the number of needed attributes, and dropping `paused_msg` is just that.  It's not because the code is in some other file that it's *not* related to this change.

Reply:
My motivation was to separate the "facelift" in case some plugins require the removed attributes. But sure, it's not strong opinion.